Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Components/Pages/PagesControllerImpl.cs

```diff
@@ -1217,9 +1217,29 @@ private static string GetExternalUrlRedirection(string url)
             if (url == null)
             {
                 return null;
-            }
-
-            return url.ToLower() == "http://" ? "" : Globals.AddHTTP(url);
+            }
+
+            if (url.ToLower() == "http://")
+            {
+                return string.Empty;
+            }
+
+            if (url.StartsWith("//"))
+            {
+                return Globals.AddHTTP(url.Remove(0, 2));

```

Comment:
```suggestion
                return url;

```

`//google.com` would be a perfectly fine url (technically a Protocol-relative link or PRL), see https://en.wikipedia.org/wiki/URL#Protocol-relative_URLs
So I suggest we keep this "un-manipulated"

In the original issue @moorecreative was describing this as a relative URL but that would be a single slack as in `/MyPage` and not `//MyPage`.

Reply:
Thank you @valadas - I am testing this change now.